src/main/java/net/atos/client/zgw/zrc/ZRCClientService.java

```diff
@@ -91,7 +91,7 @@ public void createRol(final Rol<?> rol) {
      * @return Created {@link Rol}.
      */
     public Rol<?> createRol(final Rol<?> rol, final String toelichting) {
-        zgwClientHeadersFactory.setAuditToelichting("%s: %s".formatted(rol.getOmschrijving(), toelichting));
```

Comment:
I think this would break the new `ZaakHistorieTest`

Reply:
You are absolutely right, thanks. Will fix it